src/Microsoft.DotNet.Cli.Utils/CommandResolution/PackagedCommandSpecFactoryWithCliRuntime.cs

```diff
@@ -21,9 +21,41 @@ private static void AddAditionalParameters(string commandPath, IList<string> arg
         {
             if(PrefersCliRuntime(commandPath))
             {
-                arguments.Add("--fx-version");
-                arguments.Add(new Muxer().SharedFxVersion);
+                var runtimeConfigFile = Path.ChangeExtension(commandPath, FileNameSuffixes.RuntimeConfigJson);
```

Comment:
All tools have it at this point and the guidance that we provide is for people to package it when they are packaging tools.